### ISSUE TITLE
User list needs to be contained by a tablist

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4101]
## Area or Component
Manage users page (settings)

## Quick Summary
All the users in the left-hand list are in a button element with a role of `tab`, but they are not contained by a `tablist` as appropriate. 

## Accessibility Violation
Certain ARIA roles must be contained by particular parents: https://dequeuniversity.com/rules/axe/4.4/aria-required-parent?application=AxeChrome

## Action Requested
Contain the list of users in a `<tablist>` element, or change the `role` not to be `tab`.

## Screenshots (optional)
<img width="227" alt="Screen Shot 2022-07-27 at 4 44 13 PM" src="https://user-images.githubusercontent.com/80282552/181390899-2961e228-5ae7-491a-8de6-14d43351e314.png">

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1647999156&range=4:4

**Comment by emmastephenson** [comment]
dupe!
